core/CMakeLists.txt

```diff
@@ -661,7 +661,7 @@ if (UNIX)
         ARGS -D lib_fileloc=${CMAKE_CURRENT_BINARY_DIR}/${locvar_name}
              -D check_textrel=${check_textrel}
              -D check_deps=${check_deps}
-             -D check_libc=OFF
+             -D check_libc=${BUILD_PACKAGE}
```

Comment:
This was always off before because there are no glibc imports for libdynamorio.  But I guess there's no harm in having it on for BUILD_PACKAGE=ON.